backend/src/routes/agent-api.ts

```
@@ -362,6 +366,7 @@ export const agentApiRoutes = new Elysia({ name: "agent-api" })
       });
       if (job !== undefined) {
         const errorMessage = typeof jobPayload.error === "string" ? jobPayload.error : null;
+        let _agentResultTooLarge = false;
```

ESLint

[error] '_agentResultTooLarge' is never reassigned. Use 'const' instead.

(prefer-const)